customization/bind-dirs.md

```diff
@@ -89,7 +89,7 @@ For example, if you wanted to make `/var/lib/tor` non-persistant in `sys-whonix`
 binds=( "${binds[@]/'/var/lib/tor'}" )
 ~~~
 
-(Editing `/usr/lib/qubes-bind-dirs.d/40_qubes-whonix.conf` directly is recommended against, since such changes get lost when that file is changed in the package on upgrades.)
+(Editing `/usr/lib/qubes-bind-dirs.d/40_qubes-whonix.conf` directly is not recommended, since such changes get lost when that file is changed in the package on upgrades.)
```

Comment:
It's not really wrong, but it's odd to put at the end of a sentence/fragment, and more common in past tense e.g:

"she recommended against the idea of merging this pull request"

If you wanted to add some weight you could change it to

"Editing... directly is **strongly discouraged**"